docs-conceptual/azps-3.8.0/install-az-ps.md

```diff
@@ -8,41 +8,52 @@ ms.date: 02/26/2020
 
 # Install Azure PowerShell
 
-This article explains how to install the Azure PowerShell modules using [PowerShellGet](/powershell/scripting/gallery/installing-psget). These
-instructions work on Windows, macOS, and Linux platforms.
+This article explains how to install the Azure PowerShell modules using
+[PowerShellGet](/powershell/scripting/gallery/installing-psget). These instructions work on Windows,
+macOS, and Linux platforms.
 
 Azure PowerShell is also available in Azure [Cloud Shell](/azure/cloud-shell/overview) and is now
 preinstalled in [Docker images](azureps-in-docker.md).
 
 ## Requirements
 
-Azure PowerShell works with PowerShell 5.1 or higher on Windows, or PowerShell Core 6.x and later on
-all platforms. You should install the
-[latest version of PowerShell Core](/powershell/scripting/install/installing-powershell#powershell-core)
-available for your operating system. Azure PowerShell has no additional requirements when run on
-PowerShell Core.
+Azure PowerShell works with PowerShell 7.x and later on all platforms. It is also supported with
```

Comment:
PowerShell 6.2.4 should also be present on the list with our recommendation to use PowerShell 7.